gtfs-realtime/spec/en/reference.md

```diff
@@ -351,18 +351,26 @@ A geographic position of a vehicle.
 
 ## _message_ TripDescriptor
 
-A descriptor that identifies an instance of a GTFS trip, or all instances of a trip along a route. To specify a single trip instance, the trip_id (and if necessary, start_time) is set. If route_id is also set, then it should be same as one that the given trip corresponds to. To specify all the trips along a given route, only the route_id should be set. Note that if the trip_id is not known, then station sequence ids in TripUpdate are not sufficient, and stop_ids must be provided as well. In addition, absolute arrival/departure times must be provided.
+A descriptor that identifies a single instance of a GTFS trip. 
+
+To specify a single trip instance, the trip_id (and if necessary, start_time and start_date - see field documentation below for when these fields are required) is set. If route_id is also set, then it should be same as one that the given trip corresponds to. 
```

Comment:
We could add the valid cases that we know today:
- trip_id for a short trip instance (no other field required - we expect it to be "today", works only for trips that don't last more than 24h)
- trip_id and start_date to cover long trips as well
- trip_id (of a frequency trip) and start date and start time
- (no trip_id): route_id, direction_id, start_date, start_time.

If trip_id AND route_id are specified, they need to be compatible with each other.

Reply:
Fixed in https://github.com/MobilityData/transit/pull/26/commits/28f357824e8530b775c613836f0f577e91d8ecb6